package/src/utils/add-virtual-import.ts

```diff
@@ -76,14 +81,22 @@ export const addVirtualImport = ({
  * Creates a Vite virtual module and updates the Astro config.
  * Virtual imports are useful for passing things like config options, or data computed within the integration.
  *
+ * @param {object} options
+ * @param {string} options.name
+ * @param {string} options.content
+ * @param {updateConfig} updateConfig
```

Comment:
```suggestion
 * @param {import("astro").HookParameters<"astro:config:setup">["updateConfig"]} options.updateConfig
```

Reply:
I actually didn't know this would work 🤔